apps/catalogue/src/components/harmonization/HarmonizationCell.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <td class="text-center" :class="tableClass">
+    <i class="fa fa-fw" :class="iconClass" />
+  </td>
+</template>
+
+<script>
+export default {
+  name: "HarmonicationCell",
+  props: {
+    status,
+  },
+  computed: {
+    iconClass() {
+      switch (this.status) {
+        case "unmapped":
+          return "fa-times";
+        case "partial":
+          return "fa-percent";
+        case "complete":
+          return "fa-check";
+        default:
+          return "fa-times";
+      }
+    },
+    tableClass() {
+      switch (this.status) {
+        case "unmapped":
+          return "table-danger";
+        case "partial":
+          return "table-warning";
+        case "complete":
+          return "table-success";
+        default:
+          return "table-danger";
```

Comment:
I think this also should table-light. I think cohorts will be put off if we make a 'x' screaming red. Because it is bad marketing to be 'red'. So rather use a more neutral color.

Reply:
'red' is a theme color , not a app color